gpt4all-chat/src/database.cpp

```diff
@@ -233,10 +233,15 @@ static const QString SELECT_COUNT_CHUNKS_SQL = uR"(
 )"_s;
 
 static const QString SELECT_CHUNKS_FTS_SQL = uR"(
-    select id, bm25(chunks_fts) as score
-    from chunks_fts
+    select c.id, bm25(chunks_fts) as score
+    from chunks_fts fts
+    join chunks c on fts.id = c.id
+    join documents d on c.document_id = d.id
```

Comment:
Doesn't chunks_fts have a document_id column we can use?
```suggestion
    from chunks_fts fts
    join documents d on fts.document_id = d.id
```